### PR TITLE
Register new package S3Dicts.jl v0.5.2

### PR DESCRIPTION
**Pull request by attobot**
Repository: [seung-lab/S3Dicts.jl](https://github.com/seung-lab/S3Dicts.jl)
Release: [v0.5.2](https://github.com/seung-lab/S3Dicts.jl/releases/tag/v0.5.2)
Travis: [Travis Build Status](https://travis-ci.org/seung-lab/S3Dicts.jl/builds/257895959)
cc: @jingpengw

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

@jingpengw This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

File: S3Dicts/versions/0.5.2/requires [hunk 1]
```diff
@@ -0,0 +1,7 @@
+julia 0.5
+AWSS3
+JSON
+Retry
+Libz
+Memoize
+BigArrays
```
Comment (tkelman) [inline S3Dicts/versions/0.5.2/requires]: This package would need to be registered first before another registered package can depend on it

**Comment by jingpengw** [comment]
removed the dependency of BigArrays

**Comment by tkelman** [comment]
Should also add `AWSCore` to `REQUIRE` if it's imported directly

**Comment by jingpengw** [comment]
thanks, just added AWSCore

**Pull request merged**